<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>random_password_generator</class>
 <widget class="QWidget" name="random_password_generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Password Generator</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Random Password Generator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="pass_length_text_edit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pass_length_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter password length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="copy_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>copy.png</normaloff>copy.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
